mParticle-Kochava/MPKitKochava.m

```diff
@@ -130,27 +130,30 @@ + (void)addCustomIdentityLinks:(NSDictionary *)identityLink {
 }
 
 #pragma mark Accessors and private methods
-- (void)identityLinkCustomerId {
-    FilteredMParticleUser *user = [self currentUser];
-    if (!user || user.userIdentities.count == 0) {
-        return;
-    }
-    
-    NSMutableDictionary *identityInfo = [[NSMutableDictionary alloc] initWithCapacity:user.userIdentities.count];
-    NSString *identityKey;
-    
-    NSString *identityValue = user.userIdentities[@(MPUserIdentityCustomerId)];
-    if (identityValue) {
-        identityKey = @"CustomerId";
-        identityInfo[identityKey] = identityValue;
-    }
-    
-    for (NSString *key in identityInfo.allKeys) {
-        [KVAIdentityLink registerWithName:key identifier:identityInfo[key]];
-    }
+- (NSError *)errorWithMessage:(NSString *)message {
+    NSError *error = [NSError errorWithDomain:MPKitKochavaErrorDomain code:0 userInfo:@{MPKitKochavaErrorKey:message}];
+    return error;
+}
+
+- (void)retrieveAttributionWithCompletionHandler:(void(^)(NSDictionary *attribution))completionHandler {
+    [KVATracker.shared.attribution retrieveResultWithCompletionHandler:^(KVAAttributionResult * _Nonnull attributionResult)
+     {
+        if (!attributionResult.rawDictionary) {
+            [self->_kitApi onAttributionCompleteWithResult:nil error:[self errorWithMessage:@"Received nil attributionData from Kochava"]];
+        } else {
+            MPAttributionResult *mParticleResult = [[MPAttributionResult alloc] init];
+            mParticleResult.linkInfo = attributionResult.rawDictionary;
+            
+            [self->_kitApi onAttributionCompleteWithResult:mParticleResult error:nil];
+        }
+        
+        if (completionHandler) {
+            completionHandler(attributionResult.rawDictionary);
+        }
+    }];
 }
 
-- (void)identityLinkOtherUserIds {
+- (void)synchronize {
```

Comment:
The `synchronize` method name should be more descriptive IMO.

Something along the lines of `synchronizeIdentity` or `linkIdentities` would better describe functionality.

I see some references in the original code base (although it's highlighted as a change here) so unclear if this is an update or not.